convert/circle/internal/orbs/datadog/datadog.go

```diff
@@ -0,0 +1,100 @@
+package datadog
+
+import (
+	"fmt"
+	"strings"
+
+	circle "github.com/drone/go-convert/convert/circle/yaml"
+	harness "github.com/drone/spec/dist/go"
+)
+
+func Convert(command string, step *circle.Custom) *harness.Step {
+	switch command {
+	case "":
+		return nil // not supported
+	case "setup":
+		return convertSetup(step)
+	case "stop":
+		return convertStop(step)
+	default:
+		return nil // not supported
+	}
+}
+
+func convertSetup(step *circle.Custom) *harness.Step {
+	agentMajorVersion, _ := step.Params["agent_major_version"].(string)
+	var scriptURL string
+
+	switch agentMajorVersion {
+	case "6":
+		scriptURL = "https://s3.amazonaws.com/dd-agent/scripts/install_script_agent6.sh"
+	case "7":
+		scriptURL = "https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh"
+	default:
+		scriptURL = "https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh" // default to agent 7
+	}
+	commands := []string{
+		// Install Datadog
+		"PARAM_DD_API_KEY=$(eval echo \"\\$PARAM_DD_API_KEY\")",
+		"if [ -n \"${DD_SITE}\" ]; then",
+		"  PARAM_DD_SITE=${DD_SITE}",
+		"fi",
+		"DD_API_KEY=${PARAM_DD_API_KEY} DD_AGENT_MAJOR_VERSION=${PARAM_DD_AGENT_MAJOR_VERSION} DD_SITE=${PARAM_DD_SITE} \\",
+		"DD_HOSTNAME=\"none\" DD_INSTALL_ONLY=\"true\" DD_APM_ENABLED=\"true\" \\",
+		fmt.Sprintf("bash -c \"$(curl -L %s)\"", scriptURL),
+		// Delete Default YAML Files
+		"if [ \"$UID\" = \"0\" ]; then export SUDO=''; else export SUDO='sudo'; fi",
```

Comment:
I think we can safely remove the `$SUDO` logic. Our runners always run as root, there is no need for sudo.

Reply:
done